### PR TITLE
Feature/37 redesign footer

### PR DESCRIPTION
**Pull request by serhiidotsyk**

### CODE REVIEW

File: InternetHospitalWebUI/src/app/Components/Layout/footer/footer.component.html [hunk 1]
```diff
@@ -1,5 +1,35 @@
-<footer>
-    <div layout="row" layout-align="center center">
-        <h2>Footer of Internet Hospital</h2>
+<div class="footer" fxLayout="column" fxLayoutGap="10px" fxLayoutAlign="end" fxFlex>
+    <div class="footer-top" fxLayoutAlign="space-around center">
+        <img width="200" height="200" src="../../../../assets/img/LogoCircle.png">
```
Comment (TREN-D) [inline InternetHospitalWebUI/src/app/Components/Layout/footer/footer.component.html]: Put img width and height in sccs file

File: InternetHospitalWebUI/src/app/Components/Layout/footer/footer.component.scss [hunk 2]
```diff
@@ -1,4 +1,87 @@
-footer {
-    background-color: #f5f5f5;
-    height: 100px;
-}
\ No newline at end of file
+body {
+    margin:0;
```
Comment (TREN-D) [inline InternetHospitalWebUI/src/app/Components/Layout/footer/footer.component.scss]: Add spaces between value and property

File: InternetHospitalWebUI/src/app/Components/Layout/footer/footer.component.scss [hunk 3]
```diff
@@ -1,4 +1,87 @@
...
+        bottom: -2px;
+        height: 2px;
+        background-color: #000000;
+        width: 0;
+        transition: width 0.4s;
+    }
+}
+
+.footer-menu {
+    a {
```
Comment (TREN-D) [inline InternetHospitalWebUI/src/app/Components/Layout/footer/footer.component.scss:81]: .footer-menu already exist, put "a" inside him

File: InternetHospitalWebUI/src/app/Components/Layout/footer/footer.component.scss [hunk 4]
```diff
@@ -1,4 +1,87 @@
...
+        font-size: 16px;
+        border-bottom: 2px solid #000000;
+    }
+    p{
+        font-size: 14px;
+    }
+}
+
+.footer {   
+    a {
```
Comment (TREN-D) [inline InternetHospitalWebUI/src/app/Components/Layout/footer/footer.component.scss]: .footer already exist, put "a" inside him

File: InternetHospitalWebUI/src/app/Components/Layout/footer/footer.component.scss (hunk 2 above)
Comment (TREN-D) [reply InternetHospitalWebUI/src/app/Components/Layout/footer/footer.component.scss]: It`s better to use ctrl+a->ctrl+k -> ctrl+f to all sccs files

File: InternetHospitalWebUI/package.json [hunk 5]
```diff
@@ -29,7 +29,7 @@
     "zone.js": "~0.8.26"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "^0.8.5",
```
Comment (edward0802) [inline InternetHospitalWebUI/package.json:32]: discard changes in this file

**Review by martyniukroman — APPROVED** [review]

**Review by voznovych — APPROVED** [review]

**Review by edward0802 — APPROVED** [review]

**Pull request merged**